packages/dialtone-icons/transformSVGtoVue.cjs

```diff
@@ -0,0 +1,27 @@
+const fs = require('node:fs');
+const _ = require('lodash');
+
+(function () {
+  if (!fs.existsSync('./src/icons')) fs.mkdirSync('./src/icons');
+
+  fs.readdir('./dist/svg', function (err, fileNames) {
+    if (err) {
+      console.log(err);
+      return -1;
+    }
+
+    fileNames.forEach(function (fileName) {
+      const svgContent = fs.readFileSync(`./dist/svg/${fileName}`, 'utf8');
+      const template = fs.readFileSync('./src/IconTemplate.vue', 'utf8');
+      const iconName = 'Icon' + _.startCase(_.camelCase(fileName.split('.')[0])).replace(/ /g, '');
+
+      let result = template.replace('__SVG_CONTENT__', svgContent);
+      result = result.replace('__ICON_NAME__', iconName)
+      result = result.replace(/clip-path="url\([^)]+\)"/g, ':clip-path="`url(#${uniqueID})`"')
+      result = result.replace(/clipPath id="[^"]+"/g, 'clipPath :id="uniqueID"')
```

Comment:
Here is where we introuduce uniqueId to the svg to avoid id conflicts?

Reply:
Yes, It's a little obscure as the actual function is on the IconTemplate file, but this works perfectly.

Reply:
There is some more ids to replace, e.g in whatsapp are
fill="url(#paint0_linear_3534_78) inside `path` 
then its being used in `linearGradient id="paint0_linear_3534_78"`

Im guessing you need to replace these one too

Reply:
I have this code in the other PR maybe it is useful for you

`modifySvgIds` 

https://github.com/dialpad/dialtone/pull/130/files#diff-b33ea21eb11057157f9398fd1ed9386e9a33089f18eda2727c99e2058fe22af8

 ```
modifySvgIds (svgContent) {
      // Find all 'url(#id)' and 'id="id"' instances
      const idRegex = /url\(#([^)]+)\)|id="([^"]+)"/g;
      // Generate a unique ID
      const uniqueId = (originalId) => `${this._uid}-${originalId}`;
      // Replace all 'url(#id)' and 'id="id"' instances with 'url(#uniqueId)' and 'id="uniqueId"
      return svgContent.replace(idRegex, (match, urlId, normalId) => {
        const idToReplace = urlId || normalId;
        if (idToReplace) {
          return match.replace(idToReplace, uniqueId(idToReplace));
        }
        return match;
      });
```

Reply:
Thanks! replaced